server/lib/paypal.ts

```diff
@@ -1,20 +1,65 @@
-import paypalPayoutsSDK from '@paypal/payouts-sdk';
+import paypal from '@paypal/payouts-sdk';
+
+import { PayoutBatchDetails, PayoutRequestBody, PayoutRequestResult } from '../types/paypal';
+
+const parseError = e => {
+  try {
+    return JSON.parse(e.message).message;
```

Comment:
Looks strange, does PayPal returns an error like `{ message: { message: '___' } }`?

Reply:
The problem here is that PayPal returns the body of the error as a JSON object and the library won't parse that.